docs/zh/reference/information_schema/pipes.md

```diff
@@ -0,0 +1,20 @@
+---
+displayed_sidebar: "Chinese"
+---
+
+# pipes
+
+`pipes` 提供当前数据库或指定数据库下所有 Pipe 的详细信息。此视图自 StarRocks v3.2 版本起支持。
+
+`pipes` 提供以下字段：
+
+| **字段**      | **描述**                                                     |
+| ------------- | ------------------------------------------------------------ |
+| DATABASE_NAME | Pipe 所属数据库的名称。                                      |
+| PIPE_ID       | Pipe 的唯一 ID。                                             |
+| PIPE_NAME     | Pipe 的名称。                                                |
+| TABLE_NAME    | StarRocks 目标表的名称。格式：`<database_name>.<table_name>`。 |
+| STATE         | Pipe 的状态，包括 `RUNNING`、`FINISHED`、`SUSPENDED`、`ERROR`。 |
+| LOAD_STATUS   | Pipe 下待导入数据文件的整体状态，包括如下字段：`loadedFiles`：已导入的数据文件总个数。`loadedBytes`：已导入的数据总量，单位为字节。`loadingFiles`：正在导入的数据文件总个数。 |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/61b3c819-ddb6-4ede-a9e6-815642e67863)
These should be organized as an unordered list.